### PR TITLE
Rename vector name

### PR DESCRIPTION
**Pull request by Technoblazed**
This was renamed in the latest csgo update.

### CODE REVIEW

File: scripting/shavit-misc.sp [hunk 1]
```diff
@@ -2081,7 +2081,7 @@ public Action Shotgun_Shot(const char[] te_name, const int[] Players, int numCli
 	TE_WriteVector("m_vecOrigin", temp);
 	TE_WriteFloat("m_vecAngles[0]", TE_ReadFloat("m_vecAngles[0]"));
 	TE_WriteFloat("m_vecAngles[1]", TE_ReadFloat("m_vecAngles[1]"));
-	TE_WriteNum("m_iWeaponID", TE_ReadNum("m_iWeaponID"));
+	TE_WriteNum("m_weapon", TE_ReadNum("m_weapon"));
```
Comment (shavitush) [inline scripting/shavit-misc.sp]: `m_weapon` is CS:GO only. `m_iWeaponID` still exists in CS:S. You might want to do this instead:

```sp
if(gEV_Type == Engine_CSS)
{
	TE_WriteNum("m_iWeaponID", TE_ReadNum("m_iWeaponID"));
}

else
{
	TE_WriteNum("m_weapon", TE_ReadNum("m_weapon"));
}
```
Also, have you verified that other than getting rid of errors, using `m_weapon` still serves the purpose of the code?

Comment (Technoblazed) [inline scripting/shavit-misc.sp]: https://camo.githubusercontent.com/20176dae89f4dddb45b9b465979f4bcb5ab4f20f/68747470733a2f2f692e746563682d6e6f2e6d652f3138303831373136313132342e706e67

**Comment by Technoblazed** [comment]
Github indentation is actually tragic, my god.

**Comment by shavitush** [comment]
No worries :)

**Pull request merged**